phd-tests/runner/src/config.rs

```diff
@@ -36,8 +36,19 @@ pub struct RunOptions {
     pub propolis_server_cmd: Utf8PathBuf,
 
     /// The command to use to launch Crucible downstairs servers.
+    ///
+    /// If this is not present, then a Crucible revision will be determined
+    /// based on the current Git revision of `propolis`' dependency on the
+    /// `crucible` crate.
     #[clap(long, value_parser)]
-    pub crucible_downstairs_cmd: Option<Utf8PathBuf>,
+    crucible_downstairs_cmd: Option<Utf8PathBuf>,
+
+    /// Disable Crucible.
```

Comment:
:+1: that makes sense! my assumption was that we wanted it to, essentially, just work out of the box, but I'm fine with changing it so that downloads are always opt-in.